src/app/code/community/Flagbit/MEP/Model/Export/Entity/Product.php

```diff
@@ -1099,6 +1100,23 @@ protected function  _getGoogleMapping($item, $mapItem) {
         return $attrValue;
     }
 
+    protected function _loadProductSummaryById($id)
+    {
+        $product = Mage::getModel('catalog/product')->load($id);
```

Comment:
Product load is ugly, can we find another way to get the rating summary without a load?